core/templates/domain/statistics/PlaythroughIssueObjectFactorySpec.ts

```diff
@@ -121,9 +121,9 @@ describe('Playthrough Issue Object Factory', () => {
     };
 
     expect(() => {
-      // TS ignore is used because playthrough dict is assigned a invalid type
-      // to test errors.
-      // @ts-ignore
+      // Here is playthrough dict has invalid issue_type in order to test
```

Comment:
Please proofread this and fix it. See notes above.

Ditto below.

Reply:
done